packages/openneuro-server/src/libs/authentication/crypto.ts

```diff
@@ -1,7 +1,7 @@
 import crypto from "crypto"
 import config from "../../config"
 
-const secret = config.auth.jwt.secret
+const secret = config.auth.jwt?.secret || ""
```

Comment:
What's the purpose of this change? I think this would make more sense to keep it from being defined as an insecure string. For tests, the config can be mocked with a testing value.

Reply:
I was getting an error before while running tests. I since changed the test and forgot to remove this.